router/grpc.go

```diff
@@ -100,16 +100,27 @@ func (g *GRPCRouter) validatePrepareBeaconProposer(m proto.Message, nodeAddr com
 			return status.Error(codes.PermissionDenied, "pubkey belongs to someone else or isn't owned by a rp node")
 		}
 
-		if !bytes.Equal(expectedFeeRecipient.Bytes(), proposer.FeeRecipient) {
-			g.m.Counter("prepare_beacon_incorrect_fee_recipient").Inc()
-			// Looks like a cheater- fee recipient doesn't match expectations
-			g.Logger.Warn("prepare_beacon_proposer called with unexpected fee recipient",
-				zap.String("expected", expectedFeeRecipient.String()), zap.String("got", hex.EncodeToString(proposer.FeeRecipient)))
-			return status.Error(codes.PermissionDenied, "incorrect fee recipient")
+		// If the fee recipient matches expectations, good, move on to the next one
+		if bytes.Equal(expectedFeeRecipient.Bytes(), proposer.FeeRecipient) {
+			g.m.Counter("prepare_beacon_correct_fee_recipient").Inc()
+			metrics.ObserveValidator(nodeAddr, pubkey)
+			continue
 		}
 
-		metrics.ObserveValidator(nodeAddr, pubkey)
-		g.m.Counter("prepare_beacon_correct_fee_recipient").Inc()
+		// rETH address is a 'safe' default fee recipient, and should be allowed.
+		// However, it does indicate a misconfigured node, so log it.
+		if bytes.Equal(g.EL.REthAddress().Bytes(), proposer.FeeRecipient) {
+			g.m.Counter("prepare_beacon_reth_fee_recipient").Inc()
+			g.Logger.Warn("prepare_beacon_proposer called with rETH fee recipient",
+				zap.String("expected", expectedFeeRecipient.String()))
```

Comment:
How do you feel about adding the node address to the log entry and using that to (try to) notify NOs that they should change it?